src/components/Bounties/BountiesListing/BountiesTabItems.tsx

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { useTheme } from 'next-themes';
+import React, { FC } from 'react';
+import FilterByTags from '~src/ui-components/FilterByTags';
+import { Tabs } from '~src/ui-components/Tabs';
+import BountiesTable from './BountiesTable';
+import { EBountiesStatuses, IBountyListing } from './types/types';
+import { useRouter } from 'next/router';
+
+interface IBountiesTabItemsProps {
+	bounties: IBountyListing[];
+}
+
+const BountiesTabItems: FC<IBountiesTabItemsProps> = (props) => {
+	const { resolvedTheme: theme } = useTheme();
+	const router = useRouter();
+	const initialTabKey = router.query.status && JSON.parse(decodeURIComponent(String(router.query.status).toUpperCase()));
```

Comment:
const initialTabKey = router.query.status ? JSON.parse(decodeURIComponent(String(router.query.status))) : 'all';
 no need for active state